core/objlist.hpp

```diff
@@ -281,6 +282,30 @@ class ObjList : public ObjListBase {
         del_bitmap_.swap(tmp_bool);
     }
 
+    size_t estimated_storage_size(const double sample_rate = 0.005) {
+        const size_t sample_num = this->get_size() * sample_rate;
+        BinStream bs;
+
+        // sample
+        std::unordered_set<size_t> sample_container;
+        std::random_device rd;
+        std::mt19937 generator(rd());
```

Comment:
Can we use `std::mt19937 generator(std::random_device())` or just `std::mt19937 generator`?